test/unit/test-viewport.js

```diff
@@ -1570,9 +1570,11 @@ describe('createViewport', () => {
       let win;
       let ampDoc;
       let viewer;
+      let sandbox;
 
       beforeEach(() => {
         win = env.win;
+        sandbox = env.sandbox;
```

Comment:
We already set `window.sandbox` in the global `beforeEach()` in `_init_tests.js`. Would it make sense to use it instead? Or will this code end up using the same instance?

https://github.com/ampproject/amphtml/blob/13c141e5b4ea313f510f297e123a0020eb8673da/test/_init_tests.js#L406

Reply:
it's different instance. i think we should use different sandbox instance for each test for better isolation.

we should probably remove `window.sandbox` later. unless it has good use cases.

Reply:
`window.sandbox` was added by @choumx in #18318 because tests were failing without it. IIRC, we set up a global sandbox because not all tests were creating their own sandbox.

Reply:
got it. those tests using global sandbox are likely legacy tests that don't have proper isolations. those are the ones we should pay extra attention :-)

Reply:
+1 to removing the global sandbox. It already existed before that PR:

> Looks like it's being set in the SandboxFixture in describes.js